_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Tables - Controls
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table controls refer to elements which may be present on any table variation that allows users to interact with the data contained within a table. These controls are a collection of smaller elements such as search bars, buttons, pagination controls, action cogs, amoung others. 
+preview-image: preview-images/tables-controls.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### When to use
+{: .hxSubSectionTitle}
+The table pattern has multiple controls that are used to interact with the contents displayed within a table. Controls may be placed in three primary locations.:
+
+- [On top of the table]({{page.url}}#top)
+- [Below the table]({{page.url}}#below)
+- [Within the table itself]({{page.url}}#within)
+
+{% endcolumn %}
+
+{% column right:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+{% figure [caption:"Table snippets showing the three locations for table controls. Actual controls present depend on the specific table needs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-controls-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+
+<div id="top" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+## Table controls - top
+{: .hxSectionTitle}
+
+The following controls may be found on top of a table:
+- Primary actions
+- Secondary actions
+- Filter actions
+- Select actions
+- Sorting actions
+- Search actions
+ 
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Definition of actions on top of a table and spacing specifications. NOT TO SCALE"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-controls-top.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Filter 
+{: .hxSubSectionTitle}
+
+The filter button is a specialized button used on top of a table. It is **always** the final button in a button group. Filter buttons come in the following variations:
```

Comment:
Button, control, or action?

Reply:
I get the confusion and will need to think about how we talk about these things. See the above comment and tell me if it still doesn't make sense.